### DISCUSSION TITLE
Setup a clear and efficient documentation

### DISCUSSION THREAD

**Discussion by celian-garcia** [perses/perses#1736] [Design Discussions]
I'm starting this discussion as I sometimes struggle a bit to find the right documentation. 

Can we agree two seconds on two things : 
- The folder architecture and definitions (what shoud/shouldn't we put inside each folder)
- The getting started flow. What a user will see at the very first arrival into the website

If you have also some other questions regarding the documentation architecture, it's the right place!

**Comment by eschabell** [comment]
The user docs effort started with a clear chain of documentation naming in a single user docs folder. What happened next was the website restructure spread it around a bit and it is now unclear how it is being presented. My outline was requested to resemble something like [this workshop](https://o11y-workshops.gitlab.io/workshop-perses) but then targeting new users of Perses: 

1. [Introduction to Perses](https://o11y-workshops.gitlab.io/workshop-perses/#/4) 
2. [Installing Perses](https://o11y-workshops.gitlab.io/workshop-perses/#/5)  <<< from source and containers
3. [Exploring dashboard tooling](https://o11y-workshops.gitlab.io/workshop-perses/#/6)
4. [Exploring Perses dashboards](https://o11y-workshops.gitlab.io/workshop-perses/#/7)
5. [Building your first dashboard](https://o11y-workshops.gitlab.io/workshop-perses/#/8)
6. [Building advanced dashboard](https://o11y-workshops.gitlab.io/workshop-perses/#/9)
7. [Building advanced++ dashboards](https://o11y-workshops.gitlab.io/workshop-perses/#/10)

The naming is not necessarily the goal for the Perses user docs, but the path was the one I was following as new users and non-engineer attendees to this workshop gave positive feedback.

**Reply by eschabell** [reply]
After the long explanation and rejection of my last PR on the user docs push, I wanted to make sure this is the todo list of what is wanted (based on comment by @Nexucis):

```
Once we have a simple getting started, then we could think about writing documentation about:

- How to share variables
- How to define a datasource
- How to test a datasource
- How to define a secret and use it in a datasource
- How to migrate your Grafana dashboards
- How to assign a role to a user
- How to use the provisioning
- How to validate a dashboard in a CI/CD
```

Can we confirm this and explain where it all needs to land and some form of standardized naming of these files to indicate they are user documentation?

Everything above done in the _docs/user_guides/_?

File naming such as _user_{TOPIC}_?

Finally, it's not clear on how to engage with the ordering of these guides in the new website.

I'm trying real hard to contribute and not waste time on things not wanted...

**Comment by sjcobb** [comment]
Is Google indexing our public docs, maybe we’re blocking their crawler intentionally until they’re further along?

But SEO isn’t very good overall, I googled variations of “Perses docs” and it turned up nothing (but a Python lib docs come up first). So I checked GitHub and it linked to the demo instance directly, looked for a docs link there and couldn’t find it (even after logging in) and then finally remembered that just perses.dev is what I wanted 😅

**Comment by Nexucis** [comment]
We have improved a lot the documentations thanks for the community and the effort of the team. We will release soon the new version (v0.51) that is coming with the new website struct.

Further improvements will continue but we have now a stable version that is satisfying now.
